Sumo-Logic-Tools/Event_Based_S3_Automation/script.py

```diff
@@ -0,0 +1,265 @@
+import requests
+import boto3
+import json
+from requests.auth import HTTPBasicAuth
+from collections import defaultdict
+
+'''
+Helps in taking in the inputs required for the script to function properly.
+There are 3 things the user has to enter.
+1. Select the API endpoint. https://help.sumologic.com/APIs/General-API-Information/Sumo-Logic-Endpoints-and-Firewall-Security can help in making the selection.
+2. Enter the Sumo accessId.
+3. Enter the Sumo accessKey.
+https://help.sumologic.com/Manage/Security/Access-Keys can help with the second and third step.
+'''
+class InitialSetupHelper:
+
+	apiEndpointOrderedList = ['https://api.us2.sumologic.com/api', 'https://api.sumologic.com/api', 'https://api.au.sumologic.com/api', 'https://api.eu.sumologic.com/api', 'https://api.de.sumologic.com/api']
```

Comment:
this seems prone to aging very quickly.  I would ask the user to refer to the docs to identify their endpoint, otherwise we will need to keep updating this script with every future deployment.